docs/_docs/monitoring-metrics/metrics.adoc

```diff
@@ -24,6 +24,23 @@ You can also access them programmatically.
 
 On this page, we've collected the most useful metrics and grouped them into various common categories based on the monitoring task.
 
+[NOTE]
```

Comment:
How about having this as a chapter? That's an essential configuration step you can't miss.

== Enabling JMX for Ignite

{the content you have}